test/spec/modules/magniteAnalyticsAdapter_spec.js

```diff
@@ -160,6 +161,25 @@ const MOCK = {
     'status': 'rendered',
     getStatusCode: () => 1,
   },
+  NO_BID: {
+    'requestId': 'fakeId',
+    'seatBidId': 'fakeId',
+    'bidderCode': 'mgnipbs',
+    'transactionId': '7b10a106-89ea-4e19-bc51-9b2e970fc42a',
+    'auctionId': '99785e47-a7c8-4c8a-ae05-ef1c717a4b4d',
+    'adUnitCode': 'box',
+    getStatusCode: () => 2,
```

Comment:
For this scenario it is actually a valid "BID_RESPONSE" it just does not map to a bid

So we still want to use the regular fields a bidResponse would have.

Perhaps we could even remove this No_BID mock and just update the test below to clone the regular `MOCK.BID_RESPONSE` and just change the values as needed.

Something like:

```js
      const bidResponse = utils.deepClone(MOCK.BID_RESPONSE);
      bidResponse.requestId = null;
      bidResponse. seatBidId = '123-456';
      bidResponse. bidderCode = 'andrewBidder';
      bidResponse. cpm = 11.7;
```

And then the expect validation should be a normal use case where there is no conversion error and stuff.